py3status/module.py

```diff
@@ -219,6 +252,9 @@ def run(self):
                         result['instance'] = self.module_inst
                         result['name'] = self.module_name
 
+                    # set universal module options in result
+                    result.update(self.module_options)
```

Comment:
Imho, the universal options should override anything set by the module, because this is what the user specified and therefore is what he requested. Can you think of a usecase where this might be undesirable?

It might be worth to mention it in the docs!